docs/content/users/usage/managing-projects.md

```diff
@@ -47,62 +47,53 @@ If you’re providing the `settings.php` or `wp-config.php` and DDEV is creating
-http://d9composer.ddev.site
-http://127.0.0.1:33233
-
-MySQL/MariaDB Credentials
--------------------------
-Username: "db", Password: "db", Default database: "db"
-
-or use root credentials when needed: Username: "root", Password: "root"
-
-Database hostname and port INSIDE container: ddev-d9-db:3306
-To connect to db server inside container or in project settings files:
-mysql --host=ddev-d9-dbcomposer --user=db --password=db --database=db
-Database hostname and port from HOST: 127.0.0.1:33231
-To connect to mysql from your host machine,
-mysql --host=127.0.0.1 --port=33231 --user=db --password=db --database=db
-
-Other Services
---------------
-Mailpit (HTTPS):    https://d9composer.ddev.site:8026
-Mailpit:            http://d9composer.ddev.site:8025
-
-DDEV ROUTER STATUS: healthy
-ssh-auth status: healthy
+$ ddev describe drup
+┌──────────────────────────────────────────────────────────────────────────────────┐
+│ Project: drup ~/workspace/drup https://drup.ddev.site                            │
+│ Docker platform: orbstack                                                        │
+│ Router: traefik                                                                  │
+├──────────┬──────┬───────────────────────────────────────────┬────────────────────┤
+│ SERVICE  │ STAT │ URL/PORT                                  │ INFO               │
+├──────────┼──────┼───────────────────────────────────────────┼────────────────────┤
+│ web      │ OK   │ https://drup.ddev.site                    │ drupal PHP8.3      │
+│          │      │ InDocker: web:8025,443,80                 │ nginx-fpm          │
+│          │      │ Host: 127.0.0.1:33016,33017               │ docroot:'web'      │
+│          │      │                                           │ Perf mode: mutagen │
+│          │      │                                           │ NodeJS:20          │
+├──────────┼──────┼───────────────────────────────────────────┼────────────────────┤
+│ db       │ OK   │ InDocker: db:3306                         │ mariadb:10.11      │
+│          │      │ Host: 127.0.0.1:33018                     │ User/Pass: 'db/db' │
+│          │      │                                           │ or 'root/root'     │
+├──────────┼──────┼───────────────────────────────────────────┼────────────────────┤
+│ Mailpit  │      │ Mailpit: https://drup.ddev.site:8026      │                    │
+│          │      │ `ddev mailpit`                            │                    │
+├──────────┼──────┼───────────────────────────────────────────┼────────────────────┤
+│ All URLs │      │ https://drup.ddev.site,                   │                    │
+│          │      │ https://127.0.0.1:33016,                  │                    │
+│          │      │ http://drup.ddev.site,                    │                    │
+│          │      │ http://127.0.0.1:33017                    │                    │
+└──────────┴──────┴───────────────────────────────────────────┴────────────────────┘
```

Comment:
It looks a bit broken:

![image](https://github.com/ddev/ddev/assets/24270994/0eedb58c-4df5-4744-9298-92451854ce46)

Perhaps use `ddev config global --simple-formatting=true` for this output as well?

Reply:
Thanks, yeah I tried that first and wasn't satisfied with its readability. 

On mine it's pretty much OK, this is Chrome/macOS. I don't greatly like it though. 

<img width="763" alt="image" src="https://github.com/ddev/ddev/assets/112444/bed77aa2-762a-4947-9f88-f18e861a4437">

Reply:
Maybe I should try a screenshot or something.

Reply:
I'm going to go ahead and get this in for now, return to the looks later.